samples/type-ahead-search-adaptive-cards/csharp/README.md

```diff
@@ -0,0 +1,94 @@
+---
+page_type: sample
```

Comment:
Let's follow the correct naming convention.  Please update the folder and make sure changes are reflected in ReadMe and Deployment scripts.

samples/bot-type-ahead-search-adaptive-cards